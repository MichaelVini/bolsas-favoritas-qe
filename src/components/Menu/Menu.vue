<template>
    <div>
        <section class="menu">
            <div class="menu-elements">
                <a href=""><p>Minha conta</p></a>
                <a href="">
                    <p>Menu</p>
                    <img src="@/assets/chevron-down-solid.svg" alt="">
                </a>
            </div>
        </section>  
    </div>
</template>

<script>
export default {
    
}
</script>

<style  src="./_menu.css" scoped>


