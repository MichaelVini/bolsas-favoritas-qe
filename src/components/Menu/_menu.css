.menu {
    background-color: var(--blue-primary);
    box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px, rgba(60, 64, 67, 0.15) 0px 2px 6px 2px;
}

.menu-elements {
    max-width: 960px;
    margin: 0 auto;
    display: flex;
    justify-content: space-between;
    flex-wrap: wrap; /* quebra a linha assim assim que os itens n√£o puderem ser mais compactados*/
    color: white;
    padding: 0px 20px 0px 20px;
    font-family: Verdana, Geneva, Tahoma, sans-serif;
    font-weight: 900;
    font-size: 1.5em;
}

.menu-elements a {
    text-decoration: none;
    color: inherit;
    display: flex;
}

.menu-elements a img {
    width: 0.7em;
    margin-left: 12px;
}