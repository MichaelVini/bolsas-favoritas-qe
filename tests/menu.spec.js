import { shallowMount } from '@vue/test-utils'
import Menu from "./../src/components/Menu/Menu.vue"


describe('Menu', ()=> {
    it('should render default state', () => {
        const wrapper = shallowMount(Menu)
        expect(wrapper.html()).toMatchSnapshot()
    })
})
