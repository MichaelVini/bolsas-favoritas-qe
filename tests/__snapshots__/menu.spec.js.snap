// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should render default state 1`] = `
"<div>
  <section class=\\"menu\\">
    <div class=\\"menu-elements\\"><a href=\\"\\">
        <p>Minha conta</p>
      </a> <a href=\\"\\">
        <p>Menu</p> <img src=\\"@/assets/chevron-down-solid.svg\\" alt=\\"\\">
      </a></div>
  </section>
</div>"
`;
